.github/actions/create_mrt/action.yml

```diff
@@ -4,10 +4,14 @@ inputs:
     description: "Mobify user email"
   mobify_api_key:
     description: "Mobify user API key"
+  cwd:
+    required: false
+    default: ${{ github.workspace }}
```

Comment:
I added cwd as an input to many actions so that I can reuse them in pwa-sfra-hybrid-tests where the actions are running in specific directories for either PWA or plugin_slas